src/test/ui/type/type-parameter-names.stderr

```diff
@@ -9,8 +9,8 @@ LL | fn foo<Foo, Bar>(x: Foo) -> Bar {
 LL |     x
    |     ^ expected type parameter `Bar`, found type parameter `Foo`
    |
-   = note: expected type `Bar`
-              found type `Foo`
+   = note: expected type parameter `Bar`
```

Comment:
This feels repetitive

Reply:
We already hide the note on "trivial" cases, like two different types of integers. We can expand what qualifies as trivial to cases like this one.